crates/runestick/src/function.rs

```diff
@@ -333,6 +355,51 @@ impl fmt::Debug for Function {
     }
 }
 
+impl RawImpl {
+    /// Perform a call over the function represented by this function pointer.
+    pub fn call<A, T>(&self, args: A) -> Result<T, VmError>
+    where
+        A: Args,
```

Comment:
Since it's sync, I believe this can use `GuardedArgs` if you want to support references: https://docs.rs/runestick/latest/runestick/trait.GuardedArgs.html

Reply:
That sounds reasonable! This uses the same interface as the regular Function. Does it make sense to change the default interface as well?